index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Is this actually needed? It only affects Internet Explorer, which should be long gone.

Reply:
Removed